cmd/celestia/rpc.go

```diff
@@ -418,32 +419,41 @@ func parseSignatureForHelpstring(methodSig reflect.StructField) string {
 	return simplifiedSignature
 }
 
-func parseNamespace(param string) (namespace.ID, error) {
-	var nID []byte
-	var err error
+// parseV0NamespaceID parses a namespace ID from a base64 or hex string. The param
+// is expected to be the user-specified portion of a v0 namespace ID (i.e. the
+// last 10 bytes).
+func parseV0NamespaceID(param string) (namespace.ID, error) {
+	userBytes, err := decodeToBytes(param)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(userBytes) > appns.NamespaceVersionZeroIDSize {
+		return nil, fmt.Errorf(
+			"namespace ID %v is too large to be a v0 namespace, want <= %v bytes",
+			userBytes, appns.NamespaceVersionZeroIDSize,
+		)
+	}
+	// if the namespace ID is <= 10 bytes, left pad it with 0s
```

Comment:
[blocking] does this actually left pad? I think it right pads b/c the first test case

Reply:
thanks